cli/remotebazel/remotebazel.go

```diff
@@ -77,6 +78,7 @@ var (
 	useSystemGitCredentials = RemoteFlagset.Bool("use_system_git_credentials", false, "Whether to use github auth pre-configured on the remote runner. If false, require https and an access token for git access.")
 	runFromBranch           = RemoteFlagset.String("run_from_branch", "", "A GitHub branch to base the remote run off. If unset, the remote workspace will mirror your local workspace.")
 	runFromCommit           = RemoteFlagset.String("run_from_commit", "", "A GitHub commit SHA to base the remote run off. If unset, the remote workspace will mirror your local workspace.")
+	script                  = RemoteFlagset.String("script", "", "Shell code to run instead of a Bazel command.")
```

Comment:
Would this work with a multi-line script? That seems to be a popular use case

Ex.
`
bazel build //...
bazel test //... 
`

If so, could you add an example to the docs?

Reply:
Also @siggisim because I know you have some thoughts on the interface for supporting bash commands in the remote bazel CLI